src/Utilities/VulnerabilityRoutes.js

```diff
@@ -50,6 +50,9 @@ export const AccountStatContext = createContext({
     hasConventionalSystems: true,
     hasEdgeDevices: false
 });
+export const accountHasSystem = (isEdgeParityEnabled, hasEdgeDevices, hasConventionalSystems) => {
+    return isEdgeParityEnabled && (hasEdgeDevices || hasConventionalSystems) || hasConventionalSystems;
+};
```

Comment:
It's fine to export the function, but you need to call it properly inside the `InsightsElement`. I'd suggest doing it as it was, but instead call the function you created like this:
`const accountHasSystem = checkAccountHasSystem(isEdgeParityEnabled, hasEdgeDevices, hasConventionalSystems)`

and to rename the exported function to eg. `checkAccountHasSystem` or as you wish

Reply:
Clearly a cat ran across my keyboard when I sent up this PR. Idk how I got this right in 5 other repos and forgot 3 steps on this one. I needed some milk